src/tokens/formats/json-formats.js

```diff
@@ -0,0 +1,70 @@
+const jsonVariables = {
+  name: 'rvt/json/variables',
+  formatter: function(dictionary) {
+    const allProperties = dictionary.allProperties;
+    const properties = dictionary.properties;
+    // eslint-disable-next-line prefer-const
+    let categories = Object.keys(properties);
+
+    // Wrapper is the string that will be built into the final output
+    let wrapper = ``;
+    wrapper += `{\n`;
+
+    // Output holds the output from each category loop
+    let output = ``;
+    categories.forEach(function(item) {
+      output += `  ${JSON.stringify(item)}:[\n`;
+
+      // Set counter to check for first item in output loop
+      let counter = 0;
```

Comment:
If I understand what's going on correctly, you may be able to get away with getting rid of this counter variable if you're only using it to check if it's the first item in the loop or not. The callback function in `forEach` accepts a second `index` argument that you could check against.